packages/core/deploy/002_deploy_timer.js

```diff
@@ -0,0 +1,20 @@
+const func = async function (hre) {
+  const { deployments, getNamedAccounts, network } = hre;
+  const { deploy } = deployments;
+
+  const { deployer } = await getNamedAccounts();
+  const { live } = network;
+
+  if (live === undefined) throw new Error("Network has no live parameter");
+
+  // If live === false, don't deploy a timer.
+  if (live === false) {
+    await deploy("Timer", {
+      from: deployer,
+      args: [],
```

Comment:
I'm not super familiar with hard had deploy but do you have to provide empty arguments like this? would it work fine without this param?

Reply:
Will check the docs.

Reply:
Good point. You don't. It seems to default to empty. Removed where empty.